Automattic/jetpack
# Social: Consolidate social initial state

Based off of #38430, this PR serves as an example of how to use the consolidated initial state.

This is a minimal version of what changes we need to make to get the job done (https://github.com/Automattic/jetpack-reach/issues/135). For now it uses the new initial state only for the `useAdminUiV1` feature flag. Moving forward, we will need to update the initial state from the backend and also update the corresponding js logic.

## Proposed changes:

* Use the consolidated initial of Jetpack to render the Social Initial state
* Use the rendered Social initial state for `useAdminUiV1` feature flag on Social admin page

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to Social admin page
* Confirm that you see the "Connect accounts" button when there are no connections
* Confirm that clicking on the button opens the connections management modal
* Confirm that you see the connections list
* Confirm that nothing changes in the editor and Jetpack settings page for both Jetpack and WPCOM sites
* For WoA sites, you may need to `jetpack build plugins/wpcomsh` and `jetpack rsync wpcomsh` to `mu-plugins`
* If you use [Query Monitor](https://wordpress.org/plugins/query-monitor/) plugin (which you always should), you can check the script dependencies for Social admin page.
* Do a sanity check for simple and atomic sites to ensure all is OK.
